shared/js/toc.jsx

```diff
@@ -560,6 +563,70 @@ loop.shared.toc = (function(mozL10n) {
     }
   });
 
+  var RoomControlsView = React.createClass({
+    statics: {
+      OPEN_DELAY: 200,
+      CLOSE_DELAY: 350
+    },
+
+    propTypes: {
+      audio: React.PropTypes.object.isRequired,
+      dispatcher: React.PropTypes.instanceOf(loop.Dispatcher).isRequired,
+      participantStore: React.PropTypes.instanceOf(loop.store.ParticipantStore).isRequired,
+      roomState: React.PropTypes.string.isRequired,
+      screen: React.PropTypes.object.isRequired,
+      video: React.PropTypes.object.isRequired
+    },
+
+    getInitialState: function() {
+      return {
+        showRoomControls: this.props.roomState === ROOM_STATES.HAS_PARTICIPANTS
```

Comment:
I think we should base this on if the participants store is registering participants or not.

The activeRoomStore/WebRTC stuff is now separate, and whilst the loop-server does have an idea of presence, it doesn't get communicated to the standalone.

Hence using Firebase & the participants store is the best thing to do here.